### PR TITLE
Updates cdb.js to 3.12.8. Uses zoom instead of bbox to generate the dashboard previews

### PR DESCRIPTION
**Pull request by javierarce**
Plus updates cdb.js to `3.12.8`

### CODE REVIEW

**Comment by javierarce** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1899/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1900/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1901/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1903/))

**Comment by javierarce** [comment]
@CartoDB/frontend: can you have a look to this, please?

**Comment by javierarce** [comment]
@CartoDB/frontend: can you review this, please? The important change is in here: https://github.com/CartoDB/cartodb/commit/0b8f84b61eefc6c73f95ef54b634b6733f0e4054

File: NEWS.md [hunk 1]
```diff
@@ -25,7 +25,9 @@
 * Remove tmp and unused files [#2328](https://github.com/CartoDB/cartodb/pull/2328)
 * Open privacy dialog directly from items [#2442](https://github.com/CartoDB/cartodb/pull/2442)
 * Fixes error handling when adding an erroneous WMS URL.
+* Do not send visible=false layers for static previews
 * Add loading+error state for privacy dialog [#2484](https://github.com/CartoDB/cartodb/pull/2484)
+>>>>>>> master
```
Comment (xavijam) [inline NEWS.md]: Conflict?

File: lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js [hunk 2]
```diff
@@ -31,7 +31,11 @@ module.exports = cdb.core.View.extend({
       url = url + "?api_key=" + this.api_key;
     }
 
-    cdb.Image(url, { https: this._isHTTPS() }).size(this.width, this.height).getUrl(this._onImageCallback);
+    if (this.options.zoom) {
+      cdb.Image(url, { override_bbox: true, https: this._isHTTPS() }).zoom(this.options.zoom).size(this.width, this.height).getUrl(this._onImageCallback);
+    } else {
+      cdb.Image(url, { override_bbox: true, https: this._isHTTPS() }).size(this.width, this.height).getUrl(this._onImageCallback);
+    }
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js]: I'd create an object of data, depending of zoom or not, and then apply that data to an only one cdb.Image thing...

File: lib/assets/javascripts/cartodb/new_dashboard/maps/maps_item.js [hunk 3]
```diff
@@ -105,6 +105,7 @@ module.exports = cdb.core.View.extend({
 
     var mapCardPreview = new MapCardPreview({
       el: this.$('.js-header'),
+      zoom:    this.model.get("zoom") + 2,
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/maps/maps_item.js]: why + 2?

Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/maps/maps_item.js]: We want to zoom in a bit

File: NEWS.md (hunk 1 above)
Comment (javierarce) [inline NEWS.md]: Ups :(

File: lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js (hunk 2 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js]: Done.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1908/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1922/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1927/))

**Pull request merged**